_config.yml

```diff
@@ -880,7 +880,7 @@ vendors:
 
   # MathJax
   # mathjax: //unpkg.com/mathjax@3/es5/tex-mml-chtml.js
-  # mathjax: //cdnjs.cloudflare.com/ajax/libs/mathjax/3.0.5/es5/tex-mml-chtml.js
+  # mathjax: //cdnjs.cloudflare.com/ajax/libs/mathjax/3.0.6/es5/tex-mml-chtml.js
```

Comment:
This link returns 404:
```
curl -I https://cdnjs.cloudflare.com/ajax/libs/mathjax/3.0.6/es5/tex-mml-chtml.js
HTTP/2 404 
date: Sun, 26 Jul 2020 02:31:50 GMT
content-type: text/html; charset=utf-8
served-in-seconds: 0.001
cf-cache-status: HIT
age: 35
expires: Sun, 26 Jul 2020 06:31:50 GMT
cache-control: public, max-age=14400
cf-request-id: 042a8f67c7000092ec5dbfc200000001
expect-ct: max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
strict-transport-security: max-age=15780000; includeSubDomains
server: cloudflare
cf-ray: 5b8ab4ec7d8f92ec-SJC
alt-svc: h3-27=":443"; ma=86400, h3-28=":443"; ma=86400, h3-29=":443"; ma=86400
```

The next release of MathJax would be 3.1.0 and including breaking changes. I will open another PR to fix it.